src/Microsoft.OpenApi.Readers/OpenApiDiagnostic.cs

```diff
@@ -26,5 +26,46 @@ public class OpenApiDiagnostic : IDiagnostic
         /// Open API specification version of the document parsed.
         /// </summary>
         public OpenApiSpecVersion SpecificationVersion { get; set; }
+
+        /// <summary>
+        /// Append another set of diagnostic Errors and Warnings to this one, this may be appended from another external
+        /// document's parsing and we want to indicate which file it originated from.
+        /// </summary>
+        /// <param name="diagnosticToAdd"></param>
```

Comment:
complete the description for the parameters

Reply:
Done